styles/dataDashboard.css

```diff
@@ -267,6 +285,36 @@ textarea {
         display: flex;
         flex-direction: row;
     }
+  
+    .vizCol {
+        width: 75%;
+        padding-left: 1rem;
+        padding-bottom: 5rem;
+    }
```

Comment:
Larger viewport. We no longer want stacking but the items to sit next to each other. Reduce width to accommodate.